test/scripts/jenkins_build_kbn_tp_sample_panel_action.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+source src/dev/ci_setup/setup_env.sh
+
+cd test/plugin_functional/plugins/kbn_tp_sample_panel_action;
+if [[ ! -d "target" ]]; then
+  checks-reporter-with-killswitch "Build kbn_tp_sample_panel_action" yarn build;
```

Comment:
What is the reason for this check? Is it to just provide a quick failure if we can't build the package. (ie: node is available)

Reply:
The flaky test runner basically runs this script over and over again in parallel out of the same workspace. I have to actually run this build script [here](https://github.com/elastic/kibana/pull/48781/files#diff-198d15176accda3aae8af322689294f5R35) before everything else to ensure that multiple processes don't try to build it in parallel at the same time